### PR TITLE
Generate fake JobViews and put them into MemSQL.

### PR DESCRIPTION
**Pull request by almathew**
## What's this PR do?
Generates fake job views and adds them to MemSQL. Useful for testing analytics locally, since all QC MemSQL requires VPN access.

I tried to create a useful distribution of cities/regions/countries/job titles/etc, and my results thus far suggest that I was successful. 

Data is random and will not represent true job views. The foundations for the data were pulled from actual job views, but should have any company-identifying information stripped.
 
## Screenshots (if appropriate)
screen shot 2017-02-27 at 8 09 50 am

## Where should the reviewer start?
With a solid understanding of what a job view is, and opinions on what fake data should look like.

## How should this be manually tested?
Try running the command (`python manage.py make_data` or `dkm make_data` if you're using Docker) against your local MemSQL db. Confirm that you are satisfied with what you see in analytics.

## Any background context you want to provide?
No.

### CODE REVIEW

File: analytics/management/commands/make_data.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        view_sources = view_sources.distinct()
+
+        # Date range we're generating fake data for
+        days_to_generate = 14
+        date_range = [datetime.datetime.now() - datetime.timedelta(x)
+                      for x in range(0, days_to_generate)]
+
+        job_views = []
+        num_jobs_to_create = random.randint(num_in_use_buids * 2,
+                                            num_in_use_buids * 20)
```
Comment (darrint) [inline analytics/management/commands/make_data.py]: I think you need at least one more zero here.

**Comment by darrint** [comment]
Having `make_data_datasource` in that directory causes it to show up as a command when you run `python manage.py` or `dkm` with no arguments.

File: analytics/management/commands/make_data.py (hunk 1 above)
Comment (almathew) [inline analytics/management/commands/make_data.py]: It's the number of entries, not the number of job views per entry.
I'm happy to increase the number of zeros, but I don't think the data is quite diverse enough to support that while still being worthwhile.

Comment (darrint) [inline analytics/management/commands/make_data.py]: A few thousand means you'll have plenty of job titles in every location. I was getting only a few tens of job views running this. IMHO the data is more interesting when there are a few thousand records.

**Comment by almathew** [comment]
The datasource file has been moved, so it no longer shows up as a manage command.

File: analytics/management/commands/make_data.py (hunk 1 above)
Comment (almathew) [inline analytics/management/commands/make_data.py]: Fair request, and it doesn't make a huge difference in run time. Two zeros have been added.

**Pull request merged**